forge-gui/res/cardsfolder/upcoming/fires_of_victory.txt

```diff
@@ -0,0 +1,8 @@
+Name:Fires of Victory
+ManaCost:1 R
+Types:Instant
+K:Kicker:2 U
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBDraw | SpellDescription$ If this spell was kicked, draw a card. CARDNAME deals damage to target creature or planeswalker equal to the number of cards in your hand.
+SVar:DBDraw:DB$ Draw | NumCards$ 1 | Condition$ Kicked | ConditionDescription$ If this spell was kicked
```

Comment:
The Draw needs to be before the DealDamage

Reply:
How and why? Do you mean just switch lines 5 and 6 with no changes? Or do I write it as A:SP$ Draw with the DealDamage as the SubAbility? If so, what happens if the spell isn't kicked?

Reply:
if you do:
* Draw With Condition
* Deal Damage

then it should just work without being kicked.

@Agetian just might need to take a look later if AI works as it should